cluster/node-pools/master-default/userdata.yaml

```diff
@@ -71,6 +71,9 @@ write_files:
       shutdownGracePeriod: "120s"
       shutdownGracePeriodCriticalPods: "80s"
 {{- end }}
+      # variables are replaced on instance start-up.
```

Comment:
How does that work?

Reply:
It's done in here: https://github.bus.zalan.do/teapot/kubernetes-on-ubuntu/pull/332